de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/HUsForInventoryStrategy.java

```diff
@@ -0,0 +1,37 @@
+package de.metas.handlingunits.inventory;
+
+import java.util.stream.Stream;
+
+import de.metas.handlingunits.model.I_M_HU;
+
+/*
+ * #%L
+ * de.metas.handlingunits.base
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public interface HUsForInventoryStrategy
+{
+	Stream<I_M_HU> streamHus();
+	
+	default boolean match(final int size)
```

Comment:
Instead of this `match(size)` method which is not intention revealing at all,
why not introducing a `int getMaxLocatorsAllowed()` method.

Also pls document how this MaxLocatorsAllowed will be used, how it will influence the inventory line generation command.